CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
-# Contributing to InferGPT
+# Contributing to InferESG
 
-Welcome to InferGPT! We appreciate your interest in contributing. 
+Welcome to InferESG! We appreciate your interest in contributing.
 Before you get started, please take a moment to review this guide.
 
 ## Ways to Contribute
```

Comment:
This no longer makes sense as we have a private Jira board and a new branch naming convention. Let me check with Ollie what we should do with this. May raise a separate ticket to make sense of this later.

Reply:
We will continue to make this Open source and in theory Abrdn or other FSIs could choose to contribute towards it. That said I wouldn't worry too much about this at the moment